alicloud/service_alicloud_cs.go

```diff
@@ -368,3 +392,90 @@ func (s *CsService) ignoreTag(t cs.Tag) bool {
 	}
 	return false
 }
+
+func (s *CsService) GetPermanentToken(clusterId string) (string, error) {
+
+	describeClusterTokensResponse, err := s.client.WithCsClient(func(csClient *cs.Client) (interface{}, error) {
+		return csClient.DescribeClusterTokens(clusterId)
+	})
+	if err != nil {
+		return "", fmt.Errorf("failed to get permanent token,because of %v", err)
+	}
+
+	tokens, ok := describeClusterTokensResponse.([]*cs.ClusterTokenResponse)
+
+	if ok != true {
+		return "", fmt.Errorf("failed to parse ClusterTokenResponse of cluster %s", clusterId)
```

Comment:
Please change line 408 to `return “”, WrapErrorf(err, DefaultErrorMsg, clusterId, “DescribeClusterTokens”, AlibabaCloudSdkGoERROR) )`